pkg/controllers/podSecurity/podsecurity.go

```diff
@@ -0,0 +1,49 @@
+package podsecurity
+
+import (
+	"context"
+	"time"
+
+	"github.com/loft-sh/vcluster/pkg/util/loghelper"
+	corev1 "k8s.io/api/core/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type PodSecurityReconciler struct {
+	client.Client
+	PodSecurityStandard string
+	Log                 loghelper.Logger
+}
+
+func (r *PodSecurityReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	client := r.Client
+
+	pssKey := "pod-security.kubernetes.io/enforce"
+	pssValue := r.PodSecurityStandard
+
+	ns := &corev1.Namespace{}
+	err := client.Get(ctx, req.NamespacedName, ns)
+	if err != nil {
+		return ctrl.Result{RequeueAfter: time.Second}, err
+	}
+
+	labels := ns.GetLabels()
```

Comment:
make sure labels are non nil

Reply:
I think labels are never nil. This one is ` kubernetes.io/metadata.name=` supposed to be there always. But I'll still add nil check.